.gitignore

```diff
@@ -23,6 +23,11 @@ bin/
 
 # NetBeans specific files/directories
 .nbattrs
+**/nbactions.xml
+**/nb-configuration.xml                                                               
+**/nbproject/*
+nb-configuration.xml
```

Comment:
In git, you can only add  `nb-configuration.xml` without the asterisks
I've also aready added `nb-configuration.xml`, I think you need to rebase to the latest changes